Adding resetToRoute API

davidLeonardi opened the pull request:
This API resets the current stack to 0, and thus can be used in deeper routes to begin again with a brand new navigator stack.

I created it so that replaceRoute can be used to replace the current route and maintain the previous navigator stack, and resetToRoute can be used to replace the current route and also start over with a blank navigator stack. 

This means that "back" navigations will be gone.

davidLeonardi committed:
Adding resetToRoute API

This API resets the current stack to 0, and thus can be used in deeper routes to begin again with a brand new navigator stack

davidLeonardi committed:
Update index.js

Changed resetToRoute to use a better navigator API

Fixed some jslint warnings

davidLeonardi committed:
Merge pull request #2 from sethx/sethx-patch-1

Update index.js

MikaelCarpenter commented:
Awesome! Can you update the README to document the new feature you added. Also check out a previous [PR](https://github.com/MikaelCarpenter/gb-native-router/pull/26/files) that got rid of the drag functions. This branch still has them, so if you could just get this one up to date then its good to go! (:

davidLeonardi commented:
OK for the readme, didnt quite understand what you meant about the other PR..
Not sure i see the overlap. 
Or am i missing something?

davidLeonardi commented:
By the way, a release to npm with this and the version bump would be awesome :)

MikaelCarpenter commented on index.js:
```diff
@@ -94,14 +91,35 @@ var Router = React.createClass({
       this.customAction(opts);
     }.bind(this);
 
+    var didStartDrag = function(evt) {
```
This segment right here was removed in the PR

MikaelCarpenter commented:
but yea once we get these in I'll release to npm :) I've been having issues with npm lately that i'm still trying to resolve

davidLeonardi committed:
Update index.js

removed drag stuff as per https://github.com/MikaelCarpenter/gb-native-router/pull/26/files

MikaelCarpenter commented:
looks great man! thanks for the contribution :+1:
